modules/installation/pages/install-ibmz.adoc

```diff
@@ -90,6 +90,8 @@ The hostname should contain less than eight characters.
 
 This section covers the installation of {productname}{nbsp}{productnumber} as an extension to {sls}{nbsp}{sles-version}{nbsp}{sp-version}.
 
+// TBD: Is this correct? SUSE Manager as an extension? Doesn't SLES15SP1 on z also have the new module concept with SUSE Manager being a base product?
```

Comment:
For s390x it's also a base product AFAIK, as the producer documented at https://github.com/SUSE/spacewalk/wiki/SCC-QA-proxy#scratch40 doesn't say anything about adding it as an extension.

@Bischoff can you confirm that you tested s390x as a regular base product, installing from unified installer?